components/MenuBar.js

```diff
@@ -332,8 +332,6 @@ class MenuBar extends React.Component {
         )}
 
         {LoggedInUser &&
-        LoggedInUser.isRoot() /* Only Site admins can do that for now */ &&
-        LoggedInUser.hostsUserIsAdminOf() /* Don't show button if user isn't admin anywhere */ &&
```

Comment:
Is that an intended change?

Reply:
Holy cow no, I though I had reverted this one (was just for testing). Thanks for noticing!